app/models/concerns/foreman/thread_session.rb

```diff
@@ -47,14 +48,12 @@ def current
         end
 
         def current=(o)
-          unless o.nil? || o.is_a?(self)
-            raise(ArgumentError, "Unable to set current User, expected class '#{self}', got #{o.inspect}")
-          end
-
-          if o.is_a?(User)
-            Rails.logger.info "Current user: #{o.login} (#{o.admin? ? 'administrator' : 'regular user'})"
```

Comment:
I know, I can say we can live with it, this will disappear once we start using journald structured logger. It shows only the message by default.